web/.eslintrc

```diff
@@ -1,51 +1,92 @@
 {
+  "plugins": [
```

Comment:
most of these updates move the consolidate the eslints into one file. since the webpack stuff has been resolved this is the only eslint config we need now